client/src/api/schema/schema.ts

```diff
@@ -2331,8 +2331,8 @@ export interface components {
             /**
              * Type
              * @description The digest method used to create the checksum.
-             * The value (e.g. `sha-256`) SHOULD be listed as `Hash Name String` in the https://www.iana.org/assignments/named-information/named-information.xhtml#hash-alg[IANA Named Information Hash Algorithm Registry]. Other values MAY be used, as long as implementors are aware of the issues discussed in https://tools.ietf.org/html/rfc6920#section-9.4[RFC6920].
-             * GA4GH may provide more explicit guidance for use of non-IANA-registered algorithms in the future. Until then, if implementors do choose such an algorithm (e.g. because it's implemented by their storage provider), they SHOULD use an existing standard `type` value such as `md5`, `etag`, `crc32c`, `trunc512`, or `sha1`.
+             * The value (e.g. `sha-256`) SHOULD be listed as `Hash Name String` in the https://www.iana.org/assignments/named-information/named-information.xhtml#hash-alg[IANA Named Information Hash Algorithm Registry]. Other values MAY be used, as long as implementers are aware of the issues discussed in https://tools.ietf.org/html/rfc6920#section-9.4[RFC6920].
```

Comment:
Can we fix the source in https://github.com/ga4gh/data-repository-service-schemas/blob/93d70b1c8e1477c46cf2307ac859d4282ccd3346/openapi/components/schemas/Checksum.yaml#L15 ?

Reply:
https://github.com/ga4gh/data-repository-service-schemas/pull/399